### PR TITLE
[CLI::Gem] When using rspec, configure persistence file

### PR DESCRIPTION
**Pull request by segiddins**
aka make my life easier when working with new gems.

### CODE REVIEW

File: lib/bundler/templates/newgem/spec/spec_helper.rb.tt [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
+require "bundler/setup"
 require "<%= config[:namespaced_path] %>"
+
+RSpec.configure do |config|
+  # Enable flags like --only-failures and --next-failure
+  config.example_status_persistence_file_path = ".rspec_status"
+
+  config.expect_with :rspec do |c|
+    c.syntax = :expect
+  end
```
Comment (indirect) [inline lib/bundler/templates/newgem/spec/spec_helper.rb.tt]: this doesn't seem like configuring the persistence file...

Comment (segiddins) [inline lib/bundler/templates/newgem/spec/spec_helper.rb.tt]: oops, copied from the bundler specs. suppose I should've put it in another commit, but I do think it's a good idea to include it

Comment (segiddins) [inline lib/bundler/templates/newgem/spec/spec_helper.rb.tt]: @indirect do you want me to back this out, or what?

**Comment by indirect** [comment]
@bundlerbot r+

**Comment by segiddins** [comment]
@bundlerbot r=indirect
